book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -5,7 +5,7 @@ Amethyst, those "somethings" are called Entities, which are described by
 Components.
 
 Amethyst uses Specs for its ECS (Entity-component system), which is a parallel
-Entity-component system written in Rust. You can learn more about Specs in the
+library written in Rust. You can learn more about Specs in the
```

Comment:
Could you explain why you made this change? To me this actually makes less sense.

Reply:
I'd like to go with the rewording suggested there https://github.com/amethyst/amethyst/pull/1171#discussion_r235852714 since @udoprog seems to approve.